internal/manifest/manifest.go

```diff
@@ -11,11 +11,23 @@ type Manifest struct {
 }
 
 type Provider struct {
-	GCP     []GCP     `yaml:"gcp"`
-	Hetzner []Hetzner `yaml:"hetzner"`
-	AWS     []AWS     `yaml:"aws"`
-	OCI     []OCI     `yaml:"oci"`
-	Azure   []Azure   `yaml:"azure"`
+	GCP        []GCP        `yaml:"gcp"`
+	Hetzner    []Hetzner    `yaml:"hetzner"`
+	AWS        []AWS        `yaml:"aws"`
+	OCI        []OCI        `yaml:"oci"`
+	Azure      []Azure      `yaml:"azure"`
+	Cloudflare []Cloudflare `yaml:"cloudflare"`
+	HetznerDNS []HetznerDNS `yaml:"hetznerdns"`
+}
+
+type HetznerDNS struct {
```

Comment:
What is the difference between `Hetzner` and `HetznerDNS`?

I thought the input manifest would look like
```yaml
providers:
  hetzner:
    - name: hetzner-np
      credentials: xxxxx
    - name: hetzner-dns
      credentials: xxxxx
```

Reply:
I see, let's keep them separated then.

Reply:
We could actually do this, if we just drop the len=64 requirment, because the DNS token is 32 in length, and I would prefer this as adding another provider just raises more questions marks (at least for me)

Reply:
I am more inclined to have them separate as the Hetzner DNS token cannot create VMs, and the Hetzner Cloud token cannot create DNS records. By forcing users to have a distinction there, we might avoid issues when they use tokens for something that they are not capable of.